newrelic-integration-e2e/pkg/spec/definition.go

```diff
@@ -20,9 +19,15 @@ func (def *Definition) Validate() error {
 	return nil
 }
 
+type Agent struct {
+	Integrations map[string]string `yaml:"integrations"`
```

Comment:
wan't agent struct already defined in the agent package?

Reply:
This is the attribute Agent in the yaml. The other struct is used to configure the (infra)agent... maybe i can rename the other one